### PR TITLE
Cleanup output format warnings

### PR DESCRIPTION
**Pull request by no1wudi**

### CODE REVIEW

File: core/iwasm/compilation/aot_llvm.c [hunk 1]
```diff
@@ -2490,7 +2490,7 @@ aot_load_const_from_table(AOTCompContext *comp_ctx, LLVMValueRef base,
     switch (value_type) {
         case VALUE_TYPE_F32:
             /* Store the raw int bits of f32 const as a hex string */
-            snprintf(buf, sizeof(buf), "f32#%08X", value->i32);
+            snprintf(buf, sizeof(buf), "f32#%08" PRIx32, value->i32);
```
Comment (wenyongh) [inline core/iwasm/compilation/aot_llvm.c:2493]: Should PRI**X**32

**Review by wenyongh — COMMENTED** [review]
Hi, in our test, some compilers don't support these macros, could we add them in the end of core/shared/platform/include/platform_common.h if they are undefined:

```C
#ifndef PRId32
#define PRId32     "d"
#endif
#ifndef PRIi32
#define PRIi32     "i"
#endif
#ifndef PRIu32
#define PRIu32     "u"
#endif
#ifndef PRIx32
#define PRIx32     "x"
#endif
#ifndef PRIX32
#define PRIX32     "X"
#endif

#ifndef __PRI64_PREFIX 
#if UINTPTR_MAX == UINT64_MAX
#define __PRI64_PREFIX "l"
#define __PRIPTR_PREFIX "l"
#else
#define __PRI64_PREFIX "ll"
#define __PRIPTR_PREFIX
#endif
#endif

#ifndef PRId64
# define PRId64    __PRI64_PREFIX "d"
#endif
#ifndef PRIu64
#define PRIu64     __PRI64_PREFIX "u"
#endif
#ifndef PRIx64
#define PRIx64     __PRI64_PREFIX "x"
#endif
#ifndef PRIX64
#define PRIX64     __PRI64_PREFIX "X"
#endif
#ifndef PRIXPTR
# define PRIXPTR  __PRIPTR_PREFIX "X"
#endif
```

File: core/iwasm/compilation/aot_llvm.c (hunk 1 above)
Comment (no1wudi) [reply core/iwasm/compilation/aot_llvm.c]: Done

**Review by xujuntwt95329 — COMMENTED** [review]
LGTM

**Pull request merged**